portafly/portafly_api.yml

```
@@ -19,23 +19,101 @@ paths:
       operationId: developerAccountList
       summary: Get developer account list
     parameters:
-    - name: page
-      description: Page in the paginated list. Defaults to 1.
-      schema:
-        type: integer
-      in: query
-      required: false
-    - name: per_page
-      description: Number of results per page. Default and max is 500.
-      schema:
-        type: integer
-      in: query
-      required: false
-    - name: state
-      description: Account state.
-      schema:
-        type: string
-      in: query
+      - name: page
+        description: Page in the paginated list. Defaults to 1.
+        schema:
+          type: integer
+        in: query
+        required: false
+      - name: per_page
+        description: Number of results per page. Default and max is 500.
+        schema:
+          type: integer
+        in: query
+        required: false
+      - name: state
+        description: Account state.
+        schema:
+          type: string
+        in: query
+  /users:
+    summary: A list of user
+    description: A list of users
+    get:
+      responses:
+        "200":
+          $ref: '#/components/responses/userList'
+      operationId: userList
+      summary: Get account users list
+    parameters:
+      - name: page
+        description: Page in the paginated list. Defaults to 1.
+        schema:
+          type: integer
+        in: query
+      - name: per_page
+        description: Number of results per page. Default and max is 500.
+        schema:
+          type: integer
+        in: query
+  /accounts/invitations:
+    summary: A list of sent invitations
+    description: A list of sent invitations
+    get:
+      responses:
+        "200":
+          $ref: '#/components/responses/invitationList'
+      operationId: invitationList
+      summary: Get sent invitations list
+    parameters:
+      - name: page
+        description: Page in the paginated list. Defaults to 1.
+        schema:
+          type: integer
+        in: query
+      - name: per_page
+        description: Number of results per page. Default and max is 500.
+        schema:
+          type: integer
+        in: query
+      - name: account_id
+        description: Id of current account.
+        schema:
+          type: integer
+        in: query
+  /accounts/invitations/new:
+    summary: A new invitation request
+    description: A new invitation request
+    post:
+      responses:
+        "200":
+          $ref: '#/components/responses/invitationNew'
+      operationId: invitationNew
+      summary: Post a new invitations
+    parameters:
+      - name: email
+        description: Email of invitation.
+        schema:
+          type: string
+        in: query
+  /accounts/invitations/delete:
+    summary: A list of invitations to delete
+    description: A list of invitations to delete
+    post:
+      responses:
+        "200":
```

Probably 202 is better for this kind of operations 🤔 
```suggestion
        "202":
```